packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -112,39 +115,53 @@ export default class SdNavigationItem extends SolidElement {
     this.emit('sd-show', { cancelable: true });
   }
 
-  private calculatePaddingX(): string {
+  private get calculatePaddingX(): string {
     if (this.relaxed && this.indented) return 'pl-8 pr-4';
     if (this.relaxed) return 'px-4';
     if (this.indented) return 'pl-4';
     return '';
   }
 
+  private get tag() {
+    if (this.isAccordion && !this.isLink) {
+      return literal`summary`;
+    }
+    if (this.separated) {
+      return literal`div`;
+    }
+    if (this.isLink) {
+      return literal`a`;
+    }
+    return literal`button`;
+  }
```

Comment:
Currently our html-minifier only with `${tag}` and not `${this.tag}`. I fear that you have to put that back into the render method.